devito/builtins.py

```diff
@@ -98,7 +98,7 @@ def smooth(f, g, axis=None):
         dv.Operator(dv.Eq(f, g.avg(dims=axis)), name='smoother')()
 
 
-def gaussian_smooth(f, sigma=1, _order=4, mode='reflect'):
+def gaussian_smooth(f, sigma=1, _truncate=4.0, mode='reflect'):
```

Comment:
this needs a proper docstring? At least the Parameters

also, why `_truncate` and not `truncate` ?

Reply:
There was a reason, but I don't recall what it was and is now probably irrelevant. Will change.